app/enterprise/changelog.md

```diff
@@ -5,6 +5,128 @@ no_version: true
+* When Kong Manager is secured with OIDC, new workspaces do not include the auto-generated roles.
+* Able to enumerate usernames via the /admins endpoint.
+* Unable to install Kong Manager outside the root directory.
+
+#### Kong Developer Portal
+* The `/developers/:developer/applications` endpoint allowed applications to be created for a developer other than the one in the path.
+* The change password endpoint for developers did not require the original password.
+* User was not redirected to Login after a session timeout.
+* Unable to configure `Accept Http If Already Terminated` for Application Registration plugin. Instead, add both Application Registration and OAuth 2.0 Authentication, and set the config on the Oauth2 Authentication plugin.
+
+#### Plugins
+
+* [gRPC Gateway](/hub/kong-inc/grpc-gateway/)documentation is improved.
+
+* Response Transformer Advanced (`response-transformer-advanced`)
+  * Improved performance by not inheriting from the `BasePlugin` class.
+  * Empty arrays are correctly preserved.
+  * Prevent the plugin from throwing an error when its access handler did not get a chance to run (e.g., on short-circuited, unauthorized requests).
+  * Standardized on `allow` instead of `whitelist` to specify the parameter names that should be allowed in response JSON body.
+  * The plugin is now defensive against possible errors and nil header values.
+  * Fixed an issue where the plugin was validating the value in `config.replace.body` against the content type as defined in the Content-Type response header.
+
+* Request Transformer Advanced (`request-transformer-advanced`)
+  * Template errors are properly thrown.
+
+* Open ID Connect (`openid-connect`)
+  * Fixed DB-less to reload private key JWTS on each request.
+
+* Rate Limiting Advanced (`rate-limiting-advanced`)
+  * Fixed per service rate limiting.
+
+* LDAP Authentication Advanced (`ldap-auth-advanced`)
+  * Return a 500 when there's an error.
+  * Respond with a 401 instead of 403.
+  * Case-sensitive groups.
+  * Accept AD group names containing spaces.
+
+* Collector (`collector`, for Brain and Immunity)
+  * Fixed a bug that would make the plugin try to parse request/response body regardless of the content-type.
+
+* GraphQL Rate Limiting Advanced (`gql-rate-limiting-advanced`)
+  * Fixed configuration using transformation that breaks in hybrid mode.
+
+* Updated plugins bundled with Kong Enterprise for 2.1.x compatibility
+
+### Known Issues and Workarounds
+
+**Note: This 2.1.3.0 version includes the [2.1.0.0 (beta)](/enterprise/changelog/#known-issues-and-workarounds) known issues and workarounds.**
+
+* The [Rate Limiting Advanced](/hub/rate-limiting-advanced) plugin does not support the `cluster` strategy in hybrid mode. The `redis` strategy must be used instead.
+
+* The [Key Authentication Encryptions](hub/kong-inc/key-auth-enc/) (`key-auth-enc`) plugin does not function in hybrid deployment mode.
```

Comment:
The good news is we fixed this. You can remove the key-auth-enc bullet.

Reply:
Great -- will do! Thanks!